cmd/hub/Dockerfile

```diff
@@ -18,7 +18,7 @@ FROM alpine:latest
 RUN apk --no-cache add ca-certificates && addgroup -S hub && adduser -S hub -G hub
 USER hub
 WORKDIR /home/hub
-COPY --from=backend-builder /hub ./
+COPY --from=backend-builder /charts/artifacthub ./
```

Comment:
Could you please leave this one as `hub`? This is one of the cmd names, not the chart. Otherwise it'll fail building the containers when merged into master. Thanks!

Reply:
@tegioz That was a mass `sed`  casualty. Fixing now.